src/types/contracts/derivativeScrt/model.ts

```diff
@@ -0,0 +1,20 @@
+import { StakingDerivativesValidator } from './response';
```

Comment:
rather than using the response type, let's define our own with some additional clarity. For example, 'validator' with type string doesn't tell me much about what that means. validatorAddress is more clear that this is an account address.